examples/stable_diffusion_xl/gm/modules/embedders/clip/modeling_clip.py

```diff
@@ -343,6 +344,81 @@ def get_input_embeddings(self) -> nn.Cell:
     def set_input_embeddings(self, value):
         self.text_model.embeddings.token_embedding = value
 
+    def resize_token_embeddings(self, new_num_tokens: Optional[int] = None) -> nn.Embedding:
+        """
+        Resizes input token embeddings matrix of the `CLIPTextTransformer` if `new_num_tokens != config.vocab_size`.
+
+        Arguments:
+            new_num_tokens (`int`, *optional*):
+                The number of new tokens in the embedding matrix. Increasing the size will add newly initialized
+                vectors at the end. Reducing the size will remove vectors from the end. If not provided or `None`, just
+                returns a pointer to the input tokens `torch.nn.Embedding` module of the model without doing anything.
+
+        Return:
+            `torch.nn.Embedding`: Pointer to the input tokens Embeddings Module of the model.
+        """
+        model_embeds = self._resize_token_embeddings(new_num_tokens)
+        if new_num_tokens is None:
+            return model_embeds
+
+        # Update base model and current model config
+        self.config.vocab_size = model_embeds.embedding_table.shape[0]
+        self.vocab_size = model_embeds.embedding_table.shape[0]
```

Comment:
Please check here, it looks like `CLIPTextModel` doesn't have `self.config` and `self.vocab_size` members.

Reply:
`CLIPTextModel` has `self.config` because of line 337 that I edited. 
It does not have `self.vocab_size` until `self.resize_token_embeddings` is called. 
It makes sense to adjust the `vocab_size` after resizing, so I edited here.